### PR TITLE
add a welcome message and show some necessary  information in container

### PR DESCRIPTION
**Pull request by Mingbo-Lee**
feat:add a welcome message and show some necessary information of version when the container starts

### CODE REVIEW

File: docker/release/anolis_entrypoint.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+logger = logging.getLogger()
+logger.setLevel(logging.ERROR)
+
+print("\n")
+print('Here are some information of this container')
+print('{0:<40}  {1:<40}'.format("The version of Python:", sys.version))
+print("\nThe version of some packages")
+print('{0:<40}  {1:<40}'.format("SecretFlow:", secretflow.__version__))
+print('{0:<40}  {1:<40}'.format("SPU:", spu.__version__))
+print('{0:<40}  {1:<40}'.format("ray:", ray.__version__))
```
Comment (zhouaihui) [inline docker/release/anolis_entrypoint.py:32]: `ray:` -> `Ray:`

File: docker/release/anolis_entrypoint.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+banner=f"""
+Welcome to use
+   _____                     __  ________             
+  / ___/___  _____________  / /_/ ____/ /___ _      __
+  \__ \/ _ \/ ___/ ___/ _ \/ __/ /_  / / __ \ | /| / /
+ ___/ /  __/ /__/ /  /  __/ /_/ __/ / / /_/ / |/ |/ / 
+/____/\___/\___/_/   \___/\__/_/   /_/\____/|__/|__/  
+"""
+print(banner)
+import os
```
Comment (zhouaihui) [inline docker/release/anolis_entrypoint.py:10]: Can you sort the imports? Please refer to [imports-formatting](https://google.github.io/styleguide/pyguide.html#313-imports-formatting)

Comment (Mingbo-Lee) [reply docker/release/anolis_entrypoint.py:1]: importing tensorflow directly will bring the warning message, using the line to change the level of logging to block information of warnings before importing tensorflow
```
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
```

Comment (zhouaihui) [reply docker/release/anolis_entrypoint.py:1]: The imports are not in alphabetical order still.
`jax` && `jaxlib` should be in the front of `secretflow`

**Review by zhouaihui — APPROVED** [review]
LGTM

**Pull request merged**